### PR TITLE
Implement grpc create user

### PR DESCRIPTION
**Pull request by aguerra**
* Implement create user
* Remove go-swagger implementation
* Clean up

This change is [Reviewable](https://reviewable.io/reviews/luizalabs/teresa-api/192)

### CODE REVIEW

File: cmd/client/cmd/user.go [hunk 1]
```diff
@@ -99,12 +88,45 @@ func deleteUser(cmd *cobra.Command, args []string) {
 	fmt.Println("User deleted")
 }
 
+func createUser(cmd *cobra.Command, args []string) {
+	name, _ := cmd.Flags().GetString("name")
```
Comment (aguerra) [inline cmd/client/cmd/user.go]: copy pasta from previous PR's, I will take a look into it

File: pkg/server/user/user.go [hunk 2]
```diff
@@ -70,6 +75,28 @@ func (dbu *DatabaseOperations) Delete(email string) error {
...
+		return err
+	}
+	u.Name = name
+	u.Email = email
+	u.Password = string(hash)
+	u.IsAdmin = admin
+	if err := dbu.DB.Save(u).Error; err != nil {
+		return err
+	}
+	return nil
```
Comment (drgarcia1986) [inline pkg/server/user/user.go]: what about:
```diff
- if err := dbu.DB.Save(u).Error; err != nil {
-     return err
- }
- return nil
+ return dbu.DB.Save(u).Error
```
?

File: cmd/client/cmd/user.go (hunk 1 above)
Comment (drgarcia1986) [inline cmd/client/cmd/user.go]: I wouldn't do it now.
We can check for unexpected input errors in another PR.

File: pkg/server/user/user.go (hunk 2 above)
Comment (aguerra) [inline pkg/server/user/user.go]: Oh boy :+1:

**Pull request merged**